apps/web/examples/navbar/navbar.withCustomMenuIcon.tsx

```diff
@@ -0,0 +1,109 @@
+"use client";
+
+import { Navbar, NavbarBrand, NavbarCollapse, NavbarLink, NavbarToggle } from "flowbite-react";
+import Link from "next/link";
```

Comment:
lets not couple general examples with `Next.js`, so lets remove any `next/link` reference